db/migrate/20221216171738_value_overrides_to_non_canonical_profiles.rb

```diff
@@ -0,0 +1,13 @@
+class ValueOverridesToNonCanonicalProfiles < ActiveRecord::Migration[7.0]
+  def up
+    Profile.canonical.find_each do |canonical_profile|
+      canonical_profile.child_profiles.update_all(
+        value_overrides: canonical_profile.value_overrides
+      )
+    end
+  end
+
+  def down
+    Profile.canonical(false).update_all(value_overrides: {})
+  end
```

Comment:
I don't think this is really needed, but nothing goes wrong if you keep it I guess.

Reply:
Yeah, I figured it wouldn't hurt